.report/20260412_#176_통계_페이지_UI_전면_개선.md

```
@@ -0,0 +1,32 @@
+# 🚀 [기능개선][통계] 통계 페이지 UI 전면 개선 필요
+
+## 개요
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)